firstvoices/firstvoices/settings.py

```diff
@@ -279,11 +279,11 @@
         "SENTRY_ENVIRONMENT"
     ),  # Sends information to this environment on the dashboard
     release=os.getenv("SENTRY_RELEASE"),  # Tags information with this release version
-    traces_sample_rate=os.getenv(
-        "SENTRY_TRACES_SAMPLE_RATE", 1.0
+    traces_sample_rate=float(
+        os.getenv("SENTRY_TRACES_SAMPLE_RATE", 1.0)
     ),  # The percentage of traces to send to sentry (min 0.0, max 1.0)
-    sample_rate=os.getenv(
-        "SENTRY_ERROR_SAMPLE_RATE", 1.0
+    sample_rate=float(
+        os.getenv("SENTRY_ERROR_SAMPLE_RATE", 1.0)
     ),  # The percentage of errors to send to sentry (min 0.0, max 1.0)
```

Comment:
This looks good, but using an entire float for these tiny values was bugging me. Turns out Python has a number type with smaller precision (float is usually implemented as a double in C). So we could consider using the built-in Decimal type here. https://docs.python.org/3/library/decimal.html#decimal.Decimal